### ISSUE TITLE
Bump items in Search tab do not get processed next

### ISSUE THREAD

**Issue by plittlefield** [HaveAGitGat/Tdarr#1131]
**Describe the bug**
When I click the [  ] Bump button next to a file to be taken to the top (or front) of the queue it does not do that.

**To Reproduce**
When you have a few items in your Search tab, click on the little square icon [|] in the Bump column.

**Expected behavior**
I would like the item to be changed to Number 1 in the queue to be processed next - thereby overriding the FIFO order.

**Screenshots**
Not needed.

**Additional context**
This issue has been started after a suggestion from the Reddit post:-

https://www.reddit.com/r/Tdarr/comments/1hbqwbn/what_should_bump_do_in_the_search_tab/

Thanks,

Paully

**Comment by HaveAGitGat** [comment]
Hi I have tried this, I bumped the file a.mkv on the search tab:

![image](https://github.com/user-attachments/assets/cf80fd9a-4741-43f6-8be4-33d0c062dcba)

Then checking the queue shows it's bumped to the top:

![image](https://github.com/user-attachments/assets/bc4fd024-f6c2-4403-aa5b-6d56335a2264)


Did you mean something different?

**Comment by plittlefield** [comment]
Yes, that’s exactly what I mean … but alas that behaviour is not what happens to me.

I have just tried it again on the 4 movies on the Tdarr Search tab.

I clicked the [|] Bump icon button next to movie 3. on the page, the icon changes to a spinning circle and then (after about 20 seconds) changes to a cross icon [x] button.

But, the movie has not moved to position no 1.

2 things to note …

* The Tdarr library where these 4 movies are in has PAUSED and will resume at 1am (in about 1.5 hours UK time)
* I have tried the Bump button on a desktop browser and just now on my iPhone.

I would love to get this working like yours.

Paully

**Comment by HaveAGitGat** [comment]
Do you have this option enabled:

![Image](https://github.com/user-attachments/assets/a1fc7eab-e70c-4da3-ac85-c7ddb20d2324)

`Allow bumped files to be processed before other files.`

I assume it's this but re-open if needed thanks.

**Comment by HaveAGitGat** [comment]
I've updated this so if bumping is disabled it will show here: 

![Image](https://github.com/user-attachments/assets/e36bf948-2cc4-4917-833e-f683314ecf99)

**Comment by plittlefield** [comment]
> Do you have this option enabled:
> 
> ![Image](https://github.com/user-attachments/assets/a1fc7eab-e70c-4da3-ac85-c7ddb20d2324)
> 
> `Allow bumped files to be processed before other files.`
> 
> I assume it's this but re-open if needed thanks.



Hi

Where is this option?!?!

**Comment by HaveAGitGat** [comment]
Click the “Options” dropdown on the Tdarr tab

**Comment by plittlefield** [comment]
Ah ha, found it!

I was using my phone to look and it didn’t show at first but refreshed the page and it was revealed.

It WAS unticked so have ticked it now :)

Next time my server has a few files to work with I can test this out.

Thanks!

**Comment by HaveAGitGat** [comment]
Great :)

**Comment by plittlefield** [comment]
I tried it today on file #2 of the 2 items in the queue ( on a paused node +server) and it did not change the position on the page… it was still listed as #2 :(

What should happen?

**Comment by HaveAGitGat** [comment]
Do you have a screenshot?

**Comment by plittlefield** [comment]
Not right now, because I only have 1 movie in the queue / search tab.

**As soon as I get 2 or more I will do some screenshots again.**

I am running Tdarr in a Docker on Ubuntu Server 22.04 and did also restart the server to see if that fixed it, but no.

Also, before I forget, I mentioned this on Reddit (https://www.reddit.com/r/Tdarr/comments/1hbqwbn/what_should_bump_do_in_the_search_tab/) and wanted to paste it here for you to add to the Wish List please :)

"I’ve just had a thought and it would be a neat idea to put a priority in a file movie or TV show so that you could put it into a box and then when it comes to being processed, it would automatically put it to the top of the list just a thought."

Thanks,

Paully

**Comment by HaveAGitGat** [comment]
You mean you want to prioritize a library? You can do that with this option:

![Image](https://github.com/user-attachments/assets/b0dd2be3-4d20-4e79-b60d-fd6295518011)

Then can change priority with the arrows:

![Image](https://github.com/user-attachments/assets/67069169-beb6-426c-a460-85b0e4de3e3f)

**Comment by plittlefield** [comment]
No, not by Library but by movie.

e.g.

Overnight, about 4 movies are added to the queue from Radarr.

In Radarr, I have a new option box which could be a number box or a drop-down for 'priority'.

High
Medium
Low

If a movie comes in that's set to High then that Priority level is passed to Tdarr and the movie gets bumped automatically to the top.

I don't know, just a thought ... probably too much work but thought I would say it anyway.

:-)

Paully

PS: still only have 1 movie in my Tdarr queue right now :)
